tests/builder/Data/ByteString/Builder/Tests.hs

```diff
@@ -988,3 +989,9 @@ testsUtf8 =
   [ testBuilderConstr "charUtf8" charUtf8_list charUtf8
   , testBuilderConstr "stringUtf8" (foldMap charUtf8_list) stringUtf8
   ]
+
+testToConstr :: TestTree
+testToConstr = compareImpls "toConstr" s (showConstr . toConstr)
```

Comment:
This test does not seem to have much to do with `Builder`. Why is it here?

Reply:
@Bodigrim I wasn't sure where to put it. Would it be better in the Properties tests?

Reply:
Yes, please.

Reply:
It's moved now

Reply:
forgot to test it, give me a sec

Reply:
Ok, it passes now. My bad